### PR TITLE
Release: 9.8.0

### PR DESCRIPTION
**Pull request by Aljullu**
# Release Pull Request

This is the release pull request for WooCommerce Blocks plugin `9.8.0`.

## Changelog

---

```md
#### Enhancements

- Add `Product Meta` block. [8484](https://github.com/woocommerce/woocommerce-blocks/pull/8484) 
- Add `proceedToCheckoutButtonLabel` and `proceedToCheckoutButtonLink` filters and delete cached filters when registering new ones. [8613](https://github.com/woocommerce/woocommerce-blocks/pull/8613) 
- Add border style controls to the Mini Cart block. [8654](https://github.com/woocommerce/woocommerce-blocks/pull/8654) 
- Add support to non-AJAX add to cart in the Products block. [8532](https://github.com/woocommerce/woocommerce-blocks/pull/8532) 
- Archive template: add Store Breadcrumbs to the Product Archive templates and Archive template: Make Product Title a link by default [8508](https://github.com/woocommerce/woocommerce-blocks/pull/8508) 
- Fix mini-cart `Start shopping` button to follow the current theme styles. [8567](https://github.com/woocommerce/woocommerce-blocks/pull/8567) 
- Fix: change the "Add-to-Cart behaviour" mini-cart select to a toggle button. [8558](https://github.com/woocommerce/woocommerce-blocks/pull/8558) 
- Fix: enable global styles for the `Product Result Count` block. [8517](https://github.com/woocommerce/woocommerce-blocks/pull/8517) 
- Fix: increase the "Customer account" size icon. [8594](https://github.com/woocommerce/woocommerce-blocks/pull/8594) 
- Improve Mini Cart drawer close button alignment and make it inherit the text color [8605](https://github.com/woocommerce/woocommerce-blocks/pull/8605) 
- Made the cart and checkout view switcher compatible with List View in the editor. [8429](https://github.com/woocommerce/woocommerce-blocks/pull/8429) 
- Preload Mini Cart inner blocks frontend scripts to improve performance [8653](https://github.com/woocommerce/woocommerce-blocks/pull/8653) 
- Products block: improved UX and UI for the Product Attributes advanced filter. [8583](https://github.com/woocommerce/woocommerce-blocks/pull/8583) 
- Products: Improve spacing consistency of product elements in patterns. [8401](https://github.com/woocommerce/woocommerce-blocks/pull/8401) 
- Reduce the number of scripts needed to render a page containing the Mini Cart block [8657](https://github.com/woocommerce/woocommerce-blocks/pull/8657) [8703](https://github.com/woocommerce/woocommerce-blocks/pull/8703) 

#### Bug Fixes

- Fix 'Edit Mini Cart template part' link in WP 6.2 [8574](https://github.com/woocommerce/woocommerce-blocks/pull/8574) 
- Fix Mini Cart block having some translations missing in the editor. [8591](https://github.com/woocommerce/woocommerce-blocks/pull/8591) 
- Fix Mini Cart block inserter preview not showing the cart amount. [8516](https://github.com/woocommerce/woocommerce-blocks/pull/8516) 
- Fix: remove unnecessary class from the Mini-cart badge. [8557](https://github.com/woocommerce/woocommerce-blocks/pull/8557) 
- Fixed a bug where quickly clicking through shipping methods would cause UI to update multiple times after the final selection is made. [8498](https://github.com/woocommerce/woocommerce-blocks/pull/8498) 
- Products block: Fix attributes filters that do not update the editor preview correctly [8611](https://github.com/woocommerce/woocommerce-blocks/pull/8611) 
- Revert "Show Cart and Checkout blocks in Style Book" [8602](https://github.com/woocommerce/woocommerce-blocks/pull/8602) 

#### Technical debt

- Change the required minimum version from 6.1.1 to 6.1.0. [8649](https://github.com/woocommerce/woocommerce-blocks/pull/8649) 
- Only call wp.apiFetch.createPreloadingMiddleware() when necessary. [8647](https://github.com/woocommerce/woocommerce-blocks/pull/8647) 
```

---

### Prepared Updates

Please leave a comment on this PR with links to the following:

-   [x] Release announcement (announcement post on developer.woocommerce.com published after release).

https://developer.woocommerce.com/2023/03/14/woocommerce-blocks-9-8-0-release-notes/

**Developer Notes** - The following issues require developer notes in the release post:

https://github.com/woocommerce/woocommerce-blocks/pull/8613

-   [ ] Happiness engineering or Happiness/Support (if special instructions needed).
-   [ ] Relevant developer documentation (if applicable).

## Quality

> This section is for things related to quality around the release.

-   [x] Testing Instructions are included in this PR

https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/testing/releases/980.md

-   [ ] Any performance impacts are documented.

---

### CODE REVIEW

**Comment by Aljullu** [comment]
# Release Steps

The release pull request has been created! This checklist is a guide to follow for the remainder of the release process. You can check off each item in this list once completed.

-   [x] Checkout the release branch locally.

## Initial Preparation

-   [x] Close the milestone of the release you are going to ship. That will prevent newly approved PRs to be automatically assigned to that milestone.
-   [x] Create a milestone for the next version.
-   [x] Manually add the changelog entries of all affected PRs to `readme.txt`. (Technically, this should be an automated process, but it seems to broke recently. Please change this entry back, once the automated process works again.)
-   [x] Ensure you pull your changes from the remote, since GitHub Actions will have added new commits to the branch.
    -   [x] Check the version and date in the changelog section within `readme.txt`, e.g. `= 9.8.0 - YYYY-MM-DD =`
    -   [x] Check the changelog matches the one in the pull request description above.
-   [x] Update compatibility sections (if applicable). **Note:** Do not change the stable tag or plugin version; this is automated.
    -   [x] Update _Requires at least_, _Tested up to_, and _Requires PHP_ sections at the top of `readme.txt`. Note, this should also be the latest WordPress version available at time of release.
    -   [x] Update _Requires at least_, _Requires PHP_, _WC requires at least_, and _WC tested up to_ at the top of `woocommerce-gutenberg-products-block.php`. Note, this should include requiring the latest WP version at the time of release. For _WC requires at least_, use L1 (we publicly communicate L0 but technically support L1 to provide some space for folks to update). So this means if the current version of WooCommerce core is 5.8.0, then you'll want to put 5.7.0 here.
    -   [x] If necessary, update the value of `$minimum_wp_version` at the top of the `woocommerce-gutenberg-products-block.php` file to the latest available version of WordPress.
    -   [x] Check the minimum WP version supported by **WooCommerce Core** (you can find it in [its readme.txt - line `Requires at least`](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/readme.txt#L4)). If necessary, update it in `phpcs.xml`. It would be this line: `<config name="minimum_supported_wp_version" value="5.6" />`.
-   [x] Push above changes to the release branch.

## Create the Testing Notes

-   [x] Run `npm ci`
-   [x] Run `npm run package-plugin:deploy`. This will create a zip of the current branch build locally.
    -   Note: The zip file is functionally equivalent to what gets released except the version bump.
-   [x] Create the testing notes for the release.
    -   [x] For each pull request that belongs to the current release, grab the `User Facing Testing` notes from the PR's description. Be sure that the `Do not include in the Testing Notes is not flagged` checkbox is unchecked.
    -   [x] Add the notes to `docs/internal-developers/testing/releases`
    -   [x] Update the `docs/internal-developers/testing/releases/README.md` file index.
-   [x] Copy a link to the release zip you created earlier into the testing notes. To generate the link you can upload the zip as an attachment in a GitHub comment and then just copy the path (without publishing the comment).
-   [x] Commit and push the testing docs to the release branch.
-   [x] Smoke test built release zip using the testing instructions you created:
    -   Test in a clean environment, e.g. Jurassic.Ninja site.
    -   Test existing WooCommerce Blocks content works correctly after update (no block validation errors).
    -   Test to confirm blocks are available and work correctly in oldest supported WordPress version (e.g. 5.3).
    -   Confidence check - check blocks are available and function.
    -   Test to confirm new features/fixes are working correctly.
    -   Test any UI changes in mobile and desktop views.
    -   Smoke test – test a cross section of core functionality.
-   [x] Ask the porters of Rubik and Kirigami to test the built zip as well and to approve the PR if everything looks good.

Each porter is responsible for testing the PRs that fall under the focus of their own team. Shared functionality should be tested by both porters. This means that the Rubik porter will mostly test checkout blocks and Store API endpoints, while the Kirigami porter will test the product related blocks and Store API endpoints.

-   If all PRs are testing as expected, continue with the release.
-   If one or more PRs are not testing as expected: ping the PR authors and the porter of the relevant team and ask them if the change is a release blocker or not (you can also ping the team lead if any of them is not available). In general, if it's not a regression or there is no product/marketing reason why that PR is a blocker, all other PRs should default to not being blockers.
    -   If there are blockers: stop the release and ask the PR author to fix them. If the PR author is AFK, ping the porter of their team.
    -   If some PRs are not testing as expected but they are not blockers: revert the relevant commits, remove the changes from testing steps and changelog, open an issue (or reopen the original one) and proceed with the release.
    -   If minor issues are discovered during the testing, each team is responsible for logging them in Github.

## Update Pull Request description and get approvals

-   [x] Go through the description of the release pull request and edit it to update all the sections and checklist instructions there.

## Ensure hub is set up and you're authenticated

-   [x] Make sure you've got `hub` installed (`brew install hub`)
-   [x] Make sure `hub api user` returns JSON with information about your GitHub user account, if it doesn't:
    -   [x] Create a [GitHub access token](https://github.com/settings/tokens) with the `repo` permission.
    -   [x] Set the environment variables: `GITHUB_USERNAME` with your GitHub Username, and `GITHUB_TOKEN` with the token you just generated. (You may want to add these to `.bashrc` or the equivalent)
    -   [x] Run `hub api user` again and ensure JSON with information about your GitHub user account is returned.

## Push the button - Deploy

-   [x] Execute `npm run deploy`
    -   The script will ask you to enter the version number to tag. Please enter the version we're releasing right now. Do not publish any dev tags as a release.
    -   Note: the script automatically updates version numbers on Github (commits on your behalf).
    -   **ALERT**: This script will ask you if this release will be deployed to WordPress.org. You should answer yes for this release even if it is a pre-release.
    -   A GitHub release will automatically be created and this will trigger a workflow that automatically deploys the plugin to WordPress.org.

## If this release is deployed to WordPress.org

-   [x] An email confirmation is required before the new version will be released, so check your email in order to confirm the release.
-   [x] Edit the [GitHub release](https://github.com/woocommerce/woocommerce-gutenberg-products-block/releases) and copy changelog into the release notes. Ensure there is a release with the correct version, the one you entered above.
-   [x] The `#woo-blocks-repo` slack instance will be notified about the progress with the WordPress.org deploy. Watch for that. If anything goes wrong, an error will be reported and you can followup via the GitHub actions tab and the log for that workflow.
-   [x] After the wp.org workflow completes, confirm the following
    -   [x] Changelog, Version, and Last Updated on [WP.org plugin page](https://wordpress.org/plugins/woo-gutenberg-products-block/) is correct.
    -   [x] Confirm svn tag is correct, e.g. [9.8.0](https://plugins.svn.wordpress.org/woo-gutenberg-products-block/tags/9.8.0/)
    -   [x] Confirm [WooCommerce.com plugin page](https://woocommerce.com/products/woocommerce-gutenberg-products-block/) is updated. Note: this can take several hours, feel free to check it the following day.
    -   [x] Download zip and smoke test.
    -   [x] Test updating plugin from previous version.

## After Workflow completes

-   [x] ~Merge this pull request back into `trunk`. This may have merge conflicts needing resolved if there are any cherry-picked commits in the release branch.~ We couldn't merge this PR because it contains a commit we don't want in `trunk` (5490c333e62ae760e980ce5be5ba58b4f412f8ba), so instead I copy & pasted the rellevant changes: 09dcb3e8ba4d4bcfc022168051557da7c17e5a13 and 57e8491871c3227e4d18b1185ac3f19a6541f8a5).
-   [x] Update version on the `trunk` branch to be for the next version of the plugin and include the `dev` suffix (e.g. something like [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158)) for the next version. Be sure to update the version number in the following files:
    -   [x] `package-lock.json`
    -   [x] `package.json`
    -   [x] `readme.txt`
    -   [x] `src/Package.php`
    -   [x] `woocommerce-gutenberg-products-block.php`.
-   [x] Update the schedules p2 with the shipped date for the release (PdToLP-K-p2).
-   [x] Edit the GitHub milestone of the release you just shipped and add the current date as the due date (this is used to track ship date as well).

## Pull request in WooCommerce Core for Package update

This only needs to be done if this release is the last release of the feature plugin before code freeze in the WooCommerce core cycle. If this condition doesn't exist you can skip this section.

-   [x] Remind whoever is porter this week to audit our codebase to ensure this [experimental interface document](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md) is up to date. See Pca54o-rM-p2 for more details.
-   [x] Create a pull request for updating the package in the [WooCommerce Core Repository](https://github.com/woocommerce/woocommerce/) that [bumps the package version](https://github.com/woocommerce/woocommerce/blob/747cb6b7184ba9fdc875ab104da5839cfda8b4be/plugins/woocommerce/composer.json) for the Woo Blocks package to the version you are releasing. Reviewing and merging the PR is your team's responsibility.

    -   The content for the pull release can follow [this example](https://github.com/woocommerce/woocommerce/pull/32627).

        -   [x] Increase the version of `woocommerce/woocommerce-blocks` in the `plugins/woocommerce/composer.json` file
        -   [x] Run `composer update woocommerce/woocommerce-blocks` and make sure `composer-lock.json` was updated
        -   [x] Run `pnpm --filter=woocommerce changelog add` to create a new changelog file similar to this one [plugins/woocommerce/changelog/update-woocommerce-blocks-7.4.1](https://github.com/woocommerce/woocommerce/blob/5040a10d01896bcf40fd0ac538f2b7bc584ffe0a/plugins/woocommerce/changelog/update-woocommerce-blocks-7.4.1). The file will be auto-generated with your answers. For the _Significance_ entry we’ll always use `minor` for WC Blocks major releases:

            ```md
            Significance: minor
            Type: update

            Update WooCommerce Blocks to 7.4.0
            ```

    -   The PR description can follow [this example](https://github.com/woocommerce/woocommerce/pull/32627).
        -   It lists all the WooCommerce Blocks versions that are being included since the last version that you edited in `plugins/woocommerce/composer.json`. Each version should have a link for the `Release PR`, `Testing instructions` and `Release post` (if available).
        -   The changelog should be aggregated from all the releases included in the package bump and grouped per type: `Enhancements`, `Bug Fixes`, `Various` etc. This changelog will be used in the release notes for the WooCommerce release. That's why it should only list the PRs that have WooCoomerce Core in the WooCommerce Visibility section of their description. Don't include changes available in the feature plugin or development builds.
    -   Run through the testing checklist to ensure everything works in that branch for that package bump. **Note:** Testing should ensure any features/new blocks that are supposed to be behind feature gating for the core merge of this package update are working as expected.
    -   Testing should include completing the [Smoke testing checklist](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/internal-developers/testing/smoke-testing.md). It's up to you to verify that those tests have been done.
    -   [x] Verify and make any additional edits to the pull request description for things like: Changelog to be included with WooCommerce core, additional communication that might be needed elsewhere, additional marketing communication notes that may be needed, etc.
        -   [x] Assign the corresponding WC version milestone to the PR
        -   [x] After the checklist is complete and the testing is done, select the porter of your team to review the PR. Once approved, make sure you merge the PR.

-   [x] Make sure you join the `#woo-core-releases` Slack channel to represent Woo Blocks for the release of WooCommerce core this version is included in.
-   [x] Search the release thread of the WooCommerce core version in WooCommerce P2 (example: p6q8Tx-2gl-p2).
    -   [x] Subscribe to it, so you are aware of any news/changes.
    -   [x] Make sure you are listed as the _Blocks Package_ lead or add yourself if you aren't.

## Publish posts

-   [x] Post release announcement on [WooCommerce Developer Blog](https://developer.woocommerce.com/category/release-post/woocommerce-blocks-release-notes/).
    -   Ping porters from each team to know which changelog entries need to be highlighted. Ask them to write a short text and optionally provide a screenshot. They can use previous posts for inspiration, we usually try to highlight new features or API changes.
    -   Ensure the release notes are included in the post verbatim.
    -   Don't forget to use category `WooCommerce Blocks Release Notes` for the post.
-   [x] Add highlights to the WC core release post (do this even if the release you are doing is not merged into WC core):
    -   Check which WC core version will include the WC Blocks release you just did (reference: PdToLP-K-p2).
    -   Go to its Release Thread and search the _Feature Highlights_ comment (example: p6q8Tx-2gl-p2).
    -   Edit the linked draft post and add all highlights from the release you just did.
    -   Leave a comment under the _Feature Highlights_ comment in the release thread mentioning that you updated the draft with the features included in WC Blocks X.Y.
-   [x] Announce the release internally (`#woo-announcements` slack).
-   [x] Update user-facing documentation as needed. When the plugin is released, ensure user-facing documentation is kept up to date with new blocks and compatibility information. The dev team should update documents in collaboration with support team and WooCommerce docs guild. In particular, please review and update as needed:
    -   Are there any new blocks in this release? Ensure they have adequate user documentation.
    -   Ensure any major improvements or changes are documented.
-   [x] Update minimum supported versions (WordPress, WooCommerce Core) and other requirements where necessary, including:
    -   [WCCOM product page](https://woocommerce.com/products/woocommerce-gutenberg-products-block/)
    -   [WooCommerce blocks main documentation page](https://docs.woocommerce.com/document/woocommerce-blocks/)

**Review by kmanijak — APPROVED** [review]
Changes tested as per [testing notes](https://github.com/woocommerce/woocommerce-blocks/blob/release/9.8.0/docs/internal-developers/testing/releases/980.md).

✅

**Pull request closed without merging**
